geom/geom/inc/LinkDef.h

```diff
@@ -1,2 +1,2 @@
-#include "geom/geom/inc/LinkDef1.h"
-#include "geom/geom/inc/LinkDef2.h"
+#include "./LinkDef1.h"
```

Comment:
The "./" is not needed, please remove. If it is needed it's a bug in rootcling that we need to fix.